src/app/cluster/components/AddEditClusterModal/AddEditClusterForm.tsx

```diff
@@ -45,18 +45,38 @@ const valuesHaveUpdate = (values, currentCluster) => {
   if (!currentCluster || currentCluster.MigCluster.spec.isHostCluster) {
     return true;
   }
-
   const requireSSL = !currentCluster.MigCluster.spec.insecure;
-  const caBundle = currentCluster.MigCluster.spec.caBundle;
   const rawToken = atob(currentCluster.Secret.data.saToken);
   const existingEndpoint = currentCluster.MigCluster.spec.url;
-  const azureResourceGroup = currentCluster.MigCluster.spec.azureResourceGroup;
+
+  const getAzureResourceGroupVal = (azureResourceGroup) => {
+    if (azureResourceGroup) {
+      return azureResourceGroup.length > 0
+    } else return '';
```

Comment:
This is a weird `if...else` format. Can we use something more traditional, with a ternary operator or a standard if/else?

`return azureResourceGroup ? azureResourceGroup.lengh > 0 : '';` (Still need to fix the return type here).

Does the linter like this? We should see if we can update.

Reply:
+1 updated to ternaries